src/lib/core/backtrace.cc

```diff
@@ -354,7 +357,12 @@ __asm__ volatile(
 	"\tmov sp, x2\n"
 	/* Setup fisrst arg */
 	"\tmov x0, %0\n"
+	".cfi_remember_state\n"
+	".cfi_def_cfa sp, 16 * 10\n"
```

Comment:
Nit: I prefer to use 8 * 20 to point that it is offset of 20 saved registers.

Reply:
Fixed: changed to `8 * 20` for clarity.